### PR TITLE
[KEYCLOAK-7425][OSGi] Correctly import packages of httpclient and htt…

### PR DESCRIPTION
**Pull request by grgrzybek**
…p core

### CODE REVIEW

File: adapters/oidc/adapter-core/pom.xml
Comment (hmlnarik) [inline adapters/oidc/adapter-core/pom.xml]: Isn't ${apache.httpcomponents.version} a bit too strict?

**Comment by grgrzybek** [comment]
@vramik (cc @hmlnarik) I'll test tomorrow what's the best shape of Karaf features (org.keycloak:keycloak-osgi-features)

**Comment by grgrzybek** [comment]
This PR doesn't yet make `mvn:org.keycloak/keycloak-osgi-thirdparty/${project.version}` bundle unnecessary - I'll review the features and provide better (from versioning and integration with Fuse/Karaf perspective) version of `keycloak-adapter-core`

**Comment by hmlnarik** [comment]
@grgrzybek Just to doublecheck - would this have any impact on documentation?

- https://www.keycloak.org/docs/latest/securing_apps/index.html#_fuse_install_feature
- https://www.keycloak.org/docs/latest/securing_apps/index.html#_fuse7_install_feature

**Comment by grgrzybek** [comment]
This particular issue (KEYCLOAK-7425) won't have impact on the documentation - the feature names are not changed. This is still valid:
```
feature:install keycloak-pax-http-undertow keycloak-jaas
```

However the list of bundles installed will change.

**Review by hmlnarik — APPROVED** [review]

**Comment by grgrzybek** [comment]
but please don't merge yet!

**Comment by hmlnarik** [comment]
Can be merged now after IRC discussion with @grgrzybek.

**Comment by kaja-mohideen** [comment]
This is also applicable for 3.4.3.Final - Is there plan to fix and release 3.4.4?

**Pull request merged**
